packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -96,6 +104,7 @@ export default class ScriptTransformer {
     instrument: boolean,
     supportsDynamicImport: boolean,
     supportsStaticESM: boolean,
+    supportsTopLevelAwait: boolean,
```

Comment:
passing this around is a mess, so I think we should wait until Jest 27 and move to passing objects (which is a breaking refactor)